### ISSUE TITLE
Change: Include more proxies in the common proxies list

### ISSUE THREAD

**Issue by MrClock8163** [MrClock8163/Arma3ObjectBuilder#26]
### Feature to change

Common proxies

### Description

Add more frequently used proxy paths to the common proxies list

**Comment by m0kka** [comment]
common proxies used during vehicle imports could be

`a3\data_f\proxies\passenger_low01\Cargo01`
`a3\data_f\proxies\driver_offroad\driver`
`a3\data_f\proxies\gunner_hunter\Gunner`
`a3\data_f\proxies\gunner_hunter\Commander`

`a3\data_f\VolumeLightCar.p3d`

**Comment by MrClock8163** [comment]
As far as I'm aware `canclimb` is not a thing for Arma 3. `map=hide` does exist, but the way the JSON is currently formatted, it's not possible to have multiple value versions of the same property in the list. I'll look into changing this.

As for removing the builtin items, and moving everything to the custom JSON, I'll think about it, but as this is addon is primarily for Arma 3, I'm not sure I'll do that.

Adding a vertex group name list is a pretty good idea, I'll look into implementing it for a next version.

**Comment by MrClock8163** [comment]
> I also notice I forgot a "not" in my first sentence. My paths would not be generally useful.

I figured as much.

> Unless it came with automatic firematerial setup, i.e. one button to set up a material pointing to e.g. metalplate.rvmat

That's also an interesting suggestion, I will look into it. Please open a feature requrest issue for this, so that I don't forget it.

> named properties: for guns the whole konec hlvane and nabojnice etc. would be helpful, but it makes the menu unwieldy. (Another argument for putting it all in custom_json)

As I said, I'll look into creating a frequently used vertex group list for these. Since only the relevant items are loaded in the context specific lists, it won't be "unwieldy".

Please do open a feature request for this as well.

> One other thing: please make "force lowercase" not the default when exporting. It also applies to components and when I change s.th. in ObjectBuilder and create new Components, I get the ones with capital C and the ones without.

The `Force Lowercase` option is default on, because that's the only way to provide a standardized output. Some tools and blender setups (i.e. the ArmaRig for example) capitalize things differently than some Arma 3 Samples, and there might be differences between the Arma 3 and DayZ "usual" captialization too. So as things stand now, that option will remain default.

If this is really a big issue for you, you can create a short python script in blender as an internal data block, and overwrite your startup scene to always have this in the file. (Although I realize this is not an optimal solution from your point of view.)

```py
import bpy

bpy.ops.a3ob.export_p3d(filepath="path/to/save/to", force_lowercase=False)
```

**Comment by Spaghettieis** [comment]
Thanks for that tipp! I suppose editing 
C:\Users\XXXXX\AppData\Roaming\Blender Foundation\Blender\4.3\extensions\blender_org\Arma3ObjectBuilder\ui\import_export_p3d.py
line 305 accomplishes the same thing
```
    force_lowercase: bpy.props.BoolProperty(
        name = "Force Lowercase",
        description = "Export all paths and selection names as lowercase",
        default = False
    )
```

**Comment by MrClock8163** [comment]
> Thanks for that tipp! I suppose editing C:\Users\XXXXX\AppData\Roaming\Blender Foundation\Blender\4.3\extensions\blender_org\Arma3ObjectBuilder\ui\import_export_p3d.py line 305 accomplishes the same thing
> 
> ```
>     force_lowercase: bpy.props.BoolProperty(
>         name = "Force Lowercase",
>         description = "Export all paths and selection names as lowercase",
>         default = False
>     )
> ```

Yes, with the exception, that you'll have to re-edit a file every time you update the addon.
